x/ibc/light-clients/07-tendermint/types/upgrade.go

```diff
@@ -26,102 +25,129 @@ import (
 //   and ProofSpecs do not match parameters set by committed client
 func (cs ClientState) VerifyUpgradeAndUpdateState(
 	ctx sdk.Context, cdc codec.BinaryMarshaler, clientStore sdk.KVStore,
-	upgradedClient exported.ClientState, upgradeHeight exported.Height, proofUpgrade []byte,
+	upgradedClient exported.ClientState, upgradedConsState exported.ConsensusState,
+	lastHeight exported.Height, proofUpgradeClient, proofUpgradeConsState []byte,
```

Comment:
why is lastHeight `exported.Height`, it isn't verified as that type anywhere in the proof? The key uses `int64` for the version height. What happens after the upgrade occurs? Is the consensus state removed from the upgrade path?

Reply:
Good point, I need to clear the upgrade paths! Thanks for noticing!!